web/src/user_topics_ui.js

```diff
@@ -47,10 +47,11 @@ export function handle_topic_updates(user_topic_event) {
     }
 
     setTimeout(0, () => {
-        /* Rerender "all messages" if necessary, but defer until after
-         * the browser has rendered the DOM updates scheduled above. */
-        if (message_lists.current !== message_lists.home) {
-            message_lists.home.update_muting_and_rerender();
+        // Defer updates until the above DOM updates are finished.
```

Comment:
"Defer updates for any background-rendered messages lists until the visible one has been updated." would be a bit clearer.